dynamic-plugins/wrappers/red-hat-developer-hub-backstage-plugin-marketplace/package.json

```
@@ -29,6 +29,7 @@
     "export-dynamic:clean": "run export-dynamic --clean"
   },
   "dependencies": {
+    "@mui/material": "5.16.14",
```

I saw it is 
```suggestion
    "@mui/material": "5.17.1",
```
for other wrappers

---

Done thanks